.samples/angular-single-page-app.yaml

```diff
@@ -9,4 +9,4 @@ documentationPath: /angular-single-page-app/readme.md
 tags:
   - Angular
   - Single Page Application
-imageUrl: choreoanonymouspullable.azurecr.io/angular-spa:v0.9
+imageVersion: 1.0.0
```

Comment:
Shall we move the `imageVersion` up (before the tags)?

Reply:
imageVersion is the only optional field. so wouldn't it be better to have at the end

Reply:
Since tags is a list, better to have it at the end IMO. Just to yaml to look clean and help with readability. imageVersion being optional is not a concern I think.

Reply:
+1